tests/rptest/redpanda_cloud_tests/omb_validation_test.py

```diff
@@ -0,0 +1,351 @@
+                                           "config_profile_name",
+                                           default="tier-1-aws")
+        cloud_tier = CloudTierName(cloud_tier_str)
+        extra_rp_conf = None
+        num_brokers = None
+
+        if cloud_tier == CloudTierName.DOCKER:
+            # TODO: Bake the docker config into a higher layer that will
+            # automatically load these settings upon call to make_rp_service
+            config = AdvertisedTierConfigs[CloudTierName.DOCKER]
+            num_brokers = config.num_brokers
+            extra_rp_conf = {
+                'log_segment_size': config.segment_size,
+                'cloud_storage_cache_size': config.cloud_cache_size,
+                'kafka_connections_max': config.connections_limit,
+            }
+
+        super(OMBValidationTest,
+              self).__init__(test_ctx,
+                             node_prealloc_count=11,
+                             *args,
+                             num_brokers=num_brokers,
+                             extra_rp_conf=extra_rp_conf,
+                             cloud_tier=cloud_tier,
+                             disable_cloud_storage_diagnostics=True,
+                             **kwargs)
+
+        self.tier_config = self.redpanda.advertised_tier_config
+        if cloud_tier == CloudTierName.DOCKER:
+            si_settings = SISettings(
+                test_ctx,
+                log_segment_size=self.small_segment_size,
+                cloud_storage_cache_size=self.tier_config.cloud_cache_size,
+            )
+            self.redpanda.set_si_settings(si_settings)
+            self.s3_port = si_settings.cloud_storage_api_endpoint_port
+
+        test_ctx.logger.info(f"Cloud tier {cloud_tier}: {self.tier_config}")
+
+        self.rpk = RpkTool(self.redpanda)
+
+    def _optimal_partition_count(self) -> int:
+        tier_config = self.redpanda.advertised_tier_config
+        machine_config = tier_config.machine_type_config
+        return tier_config.num_brokers * machine_config.num_shards
+
+    @cluster(num_nodes=12)
+    def test_max_connections(self):
+        tier_config = self.redpanda.advertised_tier_config
+
+        # Constants
+        #
+
+        PRODUCER_TIMEOUT_MS = 5000
+        OMB_WORKERS = 4
+        SWARM_WORKERS = 7
+
+        # OMB parameters
+        #
+
+        producers = 10
+        consumers = 10
+        subscriptions = tier_config.egress_rate // tier_config.ingress_rate
+        warmup_duration = 1  # minutes
+        test_duration = 5  # minutes
+
+        workload = {
+            "name": "MaxConnectionsTest",
+            "topics": 1,
+            "partitions_per_topic": self._optimal_partition_count(),
+            "subscriptions_per_topic": subscriptions,
+            "consumer_per_subscription": consumers,
+            "producers_per_topic": producers,
+            "producer_rate": (tier_config.ingress_rate / 2) // (4 * KiB),
```

Comment:
`producer_rate` is the number of messages a second. 4KiB is the message size.

Reply:
Right I meant any reason we are going for 4KB? Isn't that on the larger side?

Reply:
Ah, sorry, it was mainly to avoid write-amplification in the test. This this particular workload there probably wouldn't be much difference in batch size between 1KiB and 4KiB messages though.

Reply:
We should use 1KB as that's what we've been using for all our cloud benchmarking to date.

Reply:
Changing the message sizes to 1KB.